docs/user/evnt-eventing-metrics.md

```diff
@@ -0,0 +1,28 @@
+# Kyma Eventing Metrics
```

Comment:
this doc also used to be in the `operations` folder. However, it's not really instructional. I wouldn't know what to do with the information. How can I take a look at these metrics? Can we add instructions for that? In that case, this could go into the `tutorials` folder.